src/mozanalysis/metrics/desktop.py

```diff
@@ -58,7 +58,7 @@
                     *,
                     DATE(submission_timestamp) AS submission_date,
                     environment.experiments
-                FROM `moz-fx-data-shared-prod`.telemetry.main
+                FROM `moz-fx-data-shared-prod`.telemetry_stable.main_v4
```

Comment:
The comment above should probably be updated too:
```#: DataSource: The telemetry_stable.main_v4 ping table.```

Reply:
Good catch, thanks!